package.json

```diff
@@ -45,22 +46,24 @@
     "underscore.string": "^3.2.2"
   },
   "devDependencies": {
+    "artsy-xapp": "*",
+    "backbone": "*",
+    "backbone-super-sync": "*",
     "body-parser": "*",
+    "brfs": "^1.4.3",
+    "browserify": "*",
+    "coffee-script": "^1.9.2",
+    "coffeeify": "^2.1.0",
     "cookie-parser": "*",
     "cookie-session": "*",
-    "morgan": "*",
+    "jade": "*",
     "mocha": "*",
+    "morgan": "*",
+    "rewire": "2.2.0",
+    "sharify": "*",
     "should": "*",
     "sinon": "*",
-    "jade": "*",
     "underscore": "*",
-    "backbone": "*",
-    "sharify": "*",
-    "backbone-super-sync": "*",
-    "browserify": "*",
-    "coffeeify": "*",
-    "artsy-xapp": "*",
-    "zombie": "*",
-    "rewire": "2.2.0"
+    "zombie": "*"
```

Comment:
`brfs` is new to the package, and `coffee-script` is now in dev-deps. Somewhere in the browserify + coffeeify def tree `brfs` is used but not included in their dependencies. It's re-ordered to be alphabetical.